### DISCUSSION TITLE
How do I force a Sync on Ente Auth?

### DISCUSSION THREAD

**Discussion by donldraper** [ente-io/ente#3934] [Q&A]
I have added several new items to my Android phone. How long are we expected to wait until they show up on other devices? Can you please add a manual sync or at least sync on additions or edits? Thank you! Love the product but getting nervous and not going any further until I see it sync in a timely fashion. 

UPDATE: Apparently the app will sync if you edit an entry after it is created. Edited one item on my phone and the 2 newly added items suddenly appeared on my android tablet but not on the MacOS desktop app. I then added one of the existing items in the MacOS app and when I saved the 2 newly added items on the phone appeared there as well. So syncing works but unclear what triggers it. Maybe  manual sync is an option?

**Comment by ua741** [comment]
On each client, the sync usually triggers automatically when the app is opened or resumed from background. Adding or editing any entry will also trigger a sync on that particular client.

**Comment by donldraper** [comment]
It appears that this trigger is occuring only when an existing entry is
edited. What I think is occuring is that the entries are synced to the
central server. But other clients don't know to sync until an entry is
edited on that client. Make sense?

Don Draper
***@***.***
